tutorials/networking/high_level_multiplayer.rst

```diff
@@ -376,29 +376,31 @@ have loaded the game scene.
     using Godot;
     using Godot.Collections;
```

Comment:
This is probably not needed any more, since we fully qualify when using `Godot.Collections.*`.